app/js/controllers/controller-outfit.js

```diff
@@ -117,6 +117,22 @@ angular.module('app').controller('OutfitController', ['$window', '$rootScope', '
   };
 
   /**
+   * Strip ship to D-class and no other components.
+   */
+  $scope.stripBuild = function() {
+		angular.forEach(ship.common, function(slot,i) {
```

Comment:
Minor thing, but the build will fail unless you use spaces instead of tabs.
2 spaces per tab :)